.github/workflows/perl-lint-checks.yml

```diff
@@ -0,0 +1,26 @@
+---
+name: 'Perl Lint Checks'
+
+on:
+  pull_request:
+  push:
+    branches:
+      - 'master'
+
+jobs:
+  perl-lint-checks:
+    runs-on: ubuntu-latest
+    name: "Perltidy"
+    container:
+      image: perldocker/perl-tester:5.26
```

Comment:
Can we use some version-less image? I would prefer something that needs to version bumping and I consider the risk of breakage due to automatically using a more recent version minor.

Reply:
correct